packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec

```diff
@@ -1,32 +1,36 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name fog-ovirt
+%global gem_require_name %{gem_name}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.2.5
+Version: 2.0.0
 Release: 1%{?dist}
 Summary: Module for the 'fog' gem to support Ovirt
 Group: Development/Languages
 License: MIT
 URL: https://github.com/fog/fog-ovirt
-Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby >= 2.0.0
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(fog-core)
+Requires: %{?scl_prefix}rubygem(activesupport)
 Requires: %{?scl_prefix}rubygem(fog-core)
 Requires: %{?scl_prefix}rubygem(fog-json)
 Requires: %{?scl_prefix}rubygem(fog-xml)
-Requires: %{?scl_prefix}rubygem(fog-xml)
-Requires: %{?scl_prefix}rubygem(ovirt-engine-sdk) >= 4.1.3
+Requires: %{?scl_prefix}rubygem(ovirt-engine-sdk) >= 4.3.1
```

Comment:
Repoclosure is unhappy about this

Reply:
and rightfully so, updated to include the bumped sdk gem